dev/bower.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "paper-tabs",
+  "private": true,
+  "dependencies": {
+    "core-icons": "Polymer/core-icons#master",
```

Comment:
These can move into the root "bower.json" and be set as the "devDependencies" rather than "dependencies" (which is intended for production).